Makefile

```diff
@@ -0,0 +1,177 @@
+# Orchestrator
+ORCHESTRATOR_PATH := $(shell pwd)
+
+# Bootstrapper
+OPERATOR_ADDRESS := 0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266
+BOOTSTRAP_OUTPUT_PATH := $(shell pwd)/build/bootstrap.json
+BOOTSTRAPPER_COMMIT := b0b647500c2ae3e3b0d99e345fa652989bca4726
+BOOTSTRAPPER_PATH := $(shell pwd)/madara-bootstrapper
+VERIFIER_ADDRESS := 0xe7f1725E7734CE288F8367e1Bb143E90bb3F0512
+
+# SNOS
+CAIRO_LANG_COMMIT := a86e92bfde9c171c0856d7b46580c66e004922f3
+
+# Madara
+MADARA_PATH := $(shell pwd)/madara
+MADARA_COMMIT := ad0ec8cdfcde03a8e2feef76d64a7fba7a9fb792
+MADARA_DATA_PATH := $(shell pwd)/build/madara
+
+# Pathfinder
+PATHFINDER_COMMIT := 138140e5fd967ede92806fd62de47c2e6b65712a
+PATHFINDER_DATA_PATH := $(shell pwd)/build/pathfinder
+PATHFINDER_PATH := $(shell pwd)/pathfinder
+
+# Environment file
+ENV_FILE := $(shell pwd)/.makefile.json
+PID_FILE := $(shell pwd)/.pids.json
```

Comment:
let's use the same file only? like all make file related configs will be here

Reply:
this approach is much cleaner.